v2/awsv1shim/session.go

```diff
@@ -145,6 +145,13 @@ func GetSession(ctx context.Context, awsC *awsv2.Config, c *awsbase.Config) (*se
 			})
 			r.Retryable = aws.Bool(false)
 		}
+
+		if r.IsErrorExpired() {
```

Comment:
The retryer in `awsv1shim` is only called during AWS SDK for Go v1 calls from the provider. Since this check is done after the `if r.RetryCount < constants.MaxNetworkRetryCount`, it will retry 9 times (unless the retry count is overridden in the provider) before checking if it's getting expired credentials. Is the intent to fail immediately on failed auth, or still retry?

Reply:
Fixed